Overview.src.html

```diff
@@ -4178,44 +4248,96 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
+ <li><p>Set <var>request</var>'s <span title=concept-request-initiator>initiator</span> to
  "<code title>fetch</code>".
 
- <li><p>Set <var title>r</var>'s
- <span title=concept-request-destination>destination</span> to
+ <li><p>Set <var>request</var>'s <span title=concept-request-destination>destination</span> to
  "<code title>subresource</code>".
 
+ <li><p>Let <var>stream</var> be null.
+
  <li>
   <p>Run the following <span data-anolis-spec=html>in parallel</span>:
 
-  <p><span title=concept-fetch>Fetch</span> <var title>r</var>.
+  <p><span title=concept-fetch>Fetch</span> <var>request</var>.
 
-  <p>To <span>process response</span> for <var title>response</var>, run these substeps:
+  <p>To <span>process response</span> for <var>response</var>, run these substeps:
 
   <ol>
-   <li><p>If <var title>response</var>'s <span title=concept-response-type>type</span> is
-   "<code title>error</code>", reject <var title>p</var> with a
-   <code title>TypeError</code>.
+   <li><p>If <var>response</var>'s <span title=concept-response-type>type</span> is
+   "<code title>error</code>", reject <var>p</var> with a <code title>TypeError</code> and terminate
+   these substeps.
 
-   <li><p>Otherwise, resolve <var title>p</var> with a new <code>Response</code> object
-   associated with <var title>response</var> and a new <code>Headers</code> object whose
+   <li><p>Let <var>res</var> be a new <code>Response</code> object associated with
+   <var>response</var> and a new <code>Headers</code> object whose
    <span title=concept-Headers-guard>guard</span> is "<code title>immutable</code>".
+
+   <li><p>If <var>response</var>'s <span title=concept-response-body>body</span> is null, resolve
+   <var>p</var> with <var>res</var> and terminate these substeps.
+
+   <li><p>Let <var>pull</var> be an action that <span title=concept-fetch-resume>resumes</span> the
+   ongoing fetch if it is <span title=concept-fetch-suspend>suspended</span>.
+
+   <li><p>Let <var>cancel</var> be an action that
+   <span title=concept-fetch-terminate>terminates</span> the ongoing fetch with reason
+   <i title>end-user abort</i>.
+
+   <li>
+    <p>Let <var>strategy</var> be an object. The user agent may choose any object.
+    <p class="note no-backref"><var>strategy</var> is used to control the queuing strategy of
+    <var>stream</var> constructed below.
+
+   <li>
+    <p>Set <var>stream</var> to the result of
+    <span title=concept-construct-ReadableStream>constructing</span> a
+    <code title=concept-ReadableStream>ReadableStream</code> object with <var>strategy</var>,
+    <var>pull</var> and <var>cancel</var>. If that threw an exception, run the following
+    substeps:
+
+    <ol>
+     <li><p>Reject <var>p</var> with that exception.
+
+     <li><p><span title=concept-fetch-terminate>Terminate</span> the ongoing fetch with reason
+     <i title>fatal</i>.
+    </ol>
+
+   <li>
+    <p>Otherwise, run the following substeps:
```

Comment:
This "Otherwise" is a tad confusing since the previous step doesn't start with an "If". Since the previous step ends up aborting the substeps, do we need an otherwise and substeps here?